cmake/Modules/FindLibRealSense.cmake

```diff
@@ -0,0 +1,34 @@
+###############################################################################
+# Find Intel librealsense
+#
+#     find_package(LIBREALSENSE)
+#
+# Variables defined by this module:
+#
+#  LIBREALSENSE_FOUND                 True if librealsense was found
+#  LIBREALSENSE_INCLUDE_DIRS          The location(s) of librealsense headers
+#  LIBREALSENSE_LIBRARIES             Libraries needed to use librealsense
+
+find_package(PkgConfig QUIET)
+
+# Include directories
+find_path(LIBREALSENSE_INCLUDE_DIR librealsense2/rs.h)
+set(LIBREALSENSE_INCLUDE_DIRS ${LIBREALSENSE_INCLUDE_DIR})
+mark_as_advanced(LIBREALSENSE_INCLUDE_DIRS)
+
+# Libraries
+find_library(LIBREALSENSE_LIBRARY NAMES librealsense.lib)
+find_library(LIBREALSENSE_LIBRARY_DEBUG NAMES librealsense.lib)
```

Comment:
I think the librealsense2 library name is <code>realsense2.lib</code>.
Why is it <code>librealsense2.lib</code>?

And, Please add suffix (e.g. "_d") for debug library name.